mmpose/core/post_processing/__init__.py

```diff
@@ -1,6 +1,10 @@
 # Copyright (c) OpenMMLab. All rights reserved.
 from .nms import oks_iou, oks_nms, soft_oks_nms
-from .one_euro_filter import OneEuroFilter
+#from .one_euro_filter import OneEuroFilter
```

Comment:
Delete unused comments.

Reply:
This is a previous version of One-Euro filter. I'm not sure whether to leave it or not.